crates/ra_project_model/src/sysroot.rs

```diff
@@ -23,6 +24,23 @@ struct SysrootCrateData {
     deps: Vec<SysrootCrate>,
 }
 
+fn try_find_src_path(cargo_toml: &Path) -> Result<PathBuf> {
+    if let Ok(path) = env::var("RUST_SRC_PATH") {
```

Comment:
Is `RUST_SRC_PATH` a know env variable which is used elsewhere? If not, let's prefix it wiith `RA_`

Reply:
Should we perhaps make this a config option? I *think* env var makes slightly sense in this case, as this is something you probably shouldn't configure, unless you know what you are doing, and, in that case, you probably want to easily share this for all instaces of rust-analyzer

Reply:
> Is `RUST_SRC_PATH` a know env variable which is used elsewhere? If not, let's prefix it wiith `RA_`

`RUST_SRC_PATH` is a known variable in the rust ecosystem. Racer uses it and other projects seems to use it too.

I would like to stick to an env variable. In my setup I have multiple rust versions, mostly one rust version per project. So, I can not share the src path between multiple projects.

Reply:
cc @vlad20012, might make sense to support `RUST_SRC_PATH` in IntelliJ as well.

@bkchr let's then just mention this env var in the docs!